### ISSUE TITLE
sendfile(2) should return ESPIPE when offset is not null, but inFile is not seekable

### ISSUE THREAD

**Issue by chanwit** [google/gvisor#52]
Refer to `man 2 sendfile`:
```
ERRORS
       ESPIPE offset is not NULL but the input file is not seek(2)-able.
```
sendfile syscall should return ESPIPE when the offset is not null, but inFile is not seekable.

**Comment by chanwit** [comment]
Patch to review:
https://gvisor-review.googlesource.com/c/gvisor/+/2480

**Comment by chanwit** [comment]
Fixed by 3131a6b131127e70b5e3941e3c4d292d99312fa5
